include/geom/bounding_box.h

```diff
@@ -95,7 +95,7 @@ class BoundingBox : public std::pair<Point, Point>
   bool intersects (const BoundingBox &) const;
 
   bool intersect (const BoundingBox & b) const
-    { libmesh_deprecated(); return this->intersect(b); }
+    { libmesh_deprecated(); return this->intersects(b); }
```

Comment:
@roystgnr please double-check this.  clang was warning about infinite recursion for every translation unit that included bounding_box.h...